common-design-patterns/date-modified.md

```diff
@@ -111,4 +111,42 @@ title: Date modified
     </li>
    </ul>
   </section>
+
+<div class="row">
```

Comment:
Since it looks like a bunch of this is boilerplate, with some page specific metadata, would it make sense to pull this out to a template, and some URLs in the frontmatter?